packages/boot/src/__tests__/unit/booters/datasource.booter.unit.ts

```diff
@@ -3,14 +3,11 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Application} from '@loopback/core';
-import {
-  ApplicationWithRepositories,
-  RepositoryMixin,
-} from '@loopback/repository';
-import {expect, sinon, TestSandbox} from '@loopback/testlab';
-import {resolve} from 'path';
-import {DataSourceBooter, DataSourceDefaults} from '../../..';
+import { Application } from '@loopback/core';
+import { ApplicationWithRepositories, RepositoryMixin } from '@loopback/repository';
+import { expect, sinon, TestSandbox } from '@loopback/testlab';
+import { resolve } from 'path';
+import { DataSourceBooter, DataSourceDefaults } from '../../..';
```

Comment:
The spacing looks wrong to me, I think you forgot to reformat on save? (You can also run `npm run lint:fix`).

Reply:
Hmm, there seems to be racing condition in VSCode. `Organize imports` adds spaces for import `{ ... }` while prettier should fix that.